python/test/dataset_reader_test.py

```diff
@@ -35,7 +35,7 @@ def test_dis_videos_content_ids(self):
 
     def test_disvideo_is_refvideo(self):
         l = self.dataset_reader.disvideo_is_refvideo
-        self.assertItemsEqual(indices(l, lambda e: e is True), range(9))
+        self.assertCountEqual(indices(l, lambda e: e is True), range(9))
 
```

Comment:
@silviosq these seems to be only compatible for Python3, which caused build failure. Is there a way to fix these?

Reply:
I think you can use `six` to provide backwards compatibility:

```python
import six

six.assertCountEqual(...)
```